core/naive-strategy.go

```diff
@@ -98,8 +98,13 @@ func (st *NaiveStrategy) UnrelayedPackets(src, dst *ProvableChain, sh SyncHeader
 	eg.Go(func() error {
 		return retry.Do(func() error {
 			var err error
+			now := time.Now()
 			srcPackets, err = src.QueryUnfinalizedRelayPackets(srcCtx, dst)
-			return err
+			if err != nil {
+				return err
+			}
+			logger.TimeTrack(now, "QueryUnfinalizedRelayPackets", "num_packets", len(srcPackets))
```

Comment:
Good, but we need to know which chain of src and dst is queried.
Please add `"queried_chain":"src"` or something like that.

Reply:
Added `"queried_chain"` attribute
https://github.com/hyperledger-labs/yui-relayer/pull/118/commits/4e7d461232ea176adaed0cc820fe5ba92d8ddd9c